dbt/include/snowflake/macros/adapters.sql

```diff
@@ -72,14 +72,16 @@
 )
 {% endmacro %}
 
-{% macro snowflake__create_view_as(relation, sql) -%}
+{% macro create_view_as_with_temp_flag(temporary, relation, sql) -%}
   {%- set secure = config.get('secure', default=false) -%}
   {%- set copy_grants = config.get('copy_grants', default=false) -%}
   {%- set sql_header = config.get('sql_header', none) -%}
 
   {{ sql_header if sql_header is not none }}
   create or replace {% if secure -%}
     secure
+  {%- endif %} {% if temporary -%}
+    temporary
```

Comment:
I _very_ specifically check that this is the correct relative order of `secure` and `temporary`. Snowflake after all has no mention of this in their main docs.

Reply:
🧠 very smart to confirm the order